code/modules/mob/living/silicon/robot/life.dm

```diff
@@ -153,6 +153,8 @@
 	if(!is_component_functioning("actuator"))
 		src.Paralyse(3)
 
+	if(ticker && ticker.mode.name == "nations")

```

Comment:
@Fox-McCloud oh right I forgot to post here
Ponies pointed out that aliens and simple_animals shouldn't have the HUD's
(I mean I disagree but if that's what he wants, this implementation is the best he'll get)